pandas/core/groupby/groupby.py

```diff
@@ -2452,6 +2457,21 @@ def ohlc(self) -> DataFrame:
     @doc(DataFrame.describe)
     def describe(self, **kwargs):
         with self._group_selection_context():
+            if len(self._selected_obj) == 0:
+                data = self._selected_obj
+                percentiles = list(refine_percentiles(kwargs.get("percentiles")))
```

Comment:
This looks good! But I think we should also be using the other arguments (include/exclude/datetime_is_numeric). In order to avoid the complex logic, I wonder if we can do:

For DataFrame (ndim == 2):

    self._selected_obj.describe(**kwargs).to_frame().T.iloc[:0]

For Series (ndim == 1):

    gb._selected_obj.describe().to_frame().T.iloc[:0]

The difference between no need to call unstack.

Reply:
ok. Have updated code

Reply:
Hi @rhshadrach 

If I try, 
```
if data.ndim == 1:
    return self._selected_obj.describe(**kwargs).to_frame().T.iloc[:0]
else:
    return self._selected_obj.describe(**kwargs).T.iloc[:0]
```

Result obtained 
```
Empty DataFrame
Columns: [count, unique, top, freq]
Index: []
```

The `describe_categorical_1d` function will be returned from function `select_describe_func` in `pandas/core/describe.py`

Reply:
I'm not sure I see the issue; I think your running this on `pd.DataFrame(columns=['a', 'b', 'c'])`. In this case, the column dtypes are object, and not having quantiles is the behavior we want.

```
df = DataFrame([['x', 'y', 'z']], columns=["A", "B", "C"])
print(df['B'].describe())

count     1
unique    1
top       y
freq      1
Name: B, dtype: object
```

If you specify the dtype of the empty frame as numeric, then quantiles appear:

```
df = DataFrame(columns=["A", "B", "C"], dtype=int)
print(df['B'].describe())

count    0.0
mean     NaN
std      NaN
min      NaN
25%      NaN
50%      NaN
75%      NaN
max      NaN
Name: B, dtype: float64
```

Reply:
ok. Have updated